images/macos/scripts/build/install-common-utils.sh

```
@@ -6,75 +6,71 @@
-        # Pin the version on Monterey as 2.0.x requires Xcode >=15.0 which is not available on OS12
-        xcbeautify_path=$(download_with_retry "https://raw.githubusercontent.com/Homebrew/homebrew-core/d3653e83f9c029a3fddb828ac804b07ac32f7b3b/Formula/x/xcbeautify.rb")
-        brew install "$xcbeautify_path"
+    if [[ $package == "packer" ]]; then
+        # Packer has been deprecated in Homebrew. Use tap to install Packer.
+        brew install hashicorp/tap/packer
     else
-        brew_smart_install "$package"
+        if (is_VenturaX64 || is_SonomaX64) && [[ $package == "tcl-tk@8" ]]; then
+            brew_smart_install "$package"
+            # Fix for https://github.com/actions/runner-images/issues/11074
+            ln -sf $(brew --prefix tcl-tk@8)/lib/libtcl8.6.dylib /usr/local/lib/libtcl8.6.dylib
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)